control/README.md

```diff
@@ -43,6 +43,14 @@ subxt metadata --url wss://bridge-hub-polkadot-rpc.dwellir.com -f bytes -o runti
 subxt metadata --url wss://asset-hub-polkadot-rpc.dwellir.com -f bytes -o runtimes/asset-hub-polkadot/asset-hub-metadata.bin
 ```
 
+Kusama:
+
+```shell
+subxt metadata --url wss://kusama-rpc.dwellir.com -f bytes -o runtimes/kusama/polkadot-metadata.bin
```

Comment:
Add this to ./update-runtimes.sh script

Reply:
https://github.com/Snowfork/snowbridge/pull/1431/commits/bd599e7b56bd7cb629ddf5438eca5381850d1eef